ipa-core/src/net/client/mod.rs

```diff
@@ -680,7 +692,7 @@ pub(crate) mod tests {
             .await
             .unwrap();
 
-        MpcHelperClient::resp_ok(resp).await.unwrap();
+        MpcHelperClient::<Ring>::resp_ok(resp).await.unwrap();
```

Comment:
I would move `resp_ok` out of this struct, so we don't need to spell out full name everywhere. I assume resp_ok just generates an 200 HTTP response which shouldn't depend on whether we use MPC or shard client